calcom/cal.com
# fix: Routing form accepts blank name value

This PR makes the "name" value mandatory in the Routing Form.

Fixes #9871 

/claim #9871

![Screenshot 2023-07-02 161448](https://github.com/calcom/cal.com/assets/113490540/0a6ce97c-456e-48e4-8f5c-f9bc87296c93)

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

1. Install the routing form app
2. Click to add a new routing form
3. Click to add a new routing form
4. Type a single space
5. Click on the Continue button
6. Observe
